recipes/wheel/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "wheel" %}
+{% set version = "0.29.0" %}
+{% set checksum = "555a67e4507cedee23a0deb9651e452f" %}
+{% set number = 0 %}
```

Comment:
Yeah, I was meaning to actually use it in the recipe. Though I have no problem dropping it.